### ISSUE TITLE
Compilation error on raspberry pi 4

### ISSUE THREAD

**Issue by thomasfla** [facontidavide/PlotJuggler#781]
Hi Thanks for this great tool!

I'm wanted to build plotjuggler for a raspberry pi 4 without ROS (on a debian based raspberry OS) and ran into problems that I managed to solve:

First, on the raspberry Pi 4 I had to remove  `--parallel` option as the Pi is quite limited and crashed with it.

Then I had compiling errors such as `error: 'zmq_msg_more' was not declared in this scope`
I solved them by installing libzmq3-dev `sudo apt-get install libzmq3-dev` witch replaced the libzmq-dev given in COMPILE.md

Finally, I had to run with option `--disable_opengl`

**Comment by facontidavide** [comment]
> First, on the raspberry Pi 4 I had to remove --parallel option as the Pi is quite limited and crashed with it.

Yes, this is a classic problem with memory usage, not related to this software in particular

Since you solved the issue, I will close this
